macros/variables/get_common_valid_dimension_list.sql

```diff
@@ -0,0 +1,50 @@
+{% macro get_common_valid_dimension_list(dimensions,full_set) %}
```

Comment:
Same comment in this one about calling `full_set` something more specific to what's happening in this macro, as opposed to what the variable that passes it in happens to be called, as in the prior PR